hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ClientCompatibilityNullTest_1_0.java

```diff
@@ -5905,6 +5905,39 @@ public void handle(  java.util.Collection<com.hazelcast.map.impl.querycache.even
 
 
 
+
```

Comment:
All `*Compatibility*Test_*` files are as-generated, not sure why there is this whitespace. Probably the templates have to be checked on the `hazelcast-client-protocol` side in a separate issue/PR?